packages/adapter-netlify/src/serverless.js

```diff
@@ -9,11 +9,20 @@ import { split_headers } from './headers';
 export function init(manifest) {
 	const server = new Server(manifest);
 
-	server.init({
+	let init_promise = server.init({
 		env: process.env
 	});
 
 	return async (event, context) => {
+		if (init_promise !== null) {
+			// dear Rich,
+			// should we just await it indiscriminately?
+			// sincerely,
+			// me
```

Comment:
I prefer this version! It's closer to how it currently works. Hopefully we can get rid of `init_promise` at some point soon, not sure what the timeline on netlify supporting ESM is

```suggestion
```